product-overrides.yaml

```diff
@@ -1,6 +1,6 @@
 schema_version: 1
 
-from: "rhel7:7.5-released"
+from: "rhel7:7-released"
```

Comment:
why?

Reply:
`7-released` is the latest released version of rhel7.  This means that once 7.6 is released, we'll automatically pick it up, without the need for a code change.

Reply:
Makes sense :+1: